src/plugins/chart_expressions/expression_xy/public/components/annotations.scss

```diff
@@ -0,0 +1,37 @@
+.lnsXyDecorationRotatedWrapper {
```

Comment:
I think we should change the styles prefix now for the whole file (for example from `lnsXy` to `xy`)

Same in the other scss files (I cannot comment there because the are no changes, just location (eg src/plugins/chart_expressions/expression_xy/public/components/xy_chart.scss)

Reply:
Refactored a little bit the styles.